filebeat/module/nats/log/test/test.log-expected.json

```diff
@@ -53,66 +53,258 @@
     },
     {
         "@timestamp": "2019-02-06T07:20:08.508Z",
+        "client.ip": "172.18.0.1",
+        "client.port": "38630",
         "ecs.version": "1.0.0-beta2",
         "event.dataset": "nats.log",
         "event.module": "nats",
         "fileset.name": "log",
         "input.type": "log",
         "log.level": "debug",
         "log.offset": 269,
-        "message": "172.18.0.1:38630 - cid:1 - Client connection created",
+        "message": "Client connection created",
+        "nats.log.client.id": "1",
         "process.pid": "1",
         "service.type": "nats"
     },
     {
         "@timestamp": "2019-02-06T07:20:08.510Z",
+        "client.ip": "172.18.0.1",
+        "client.port": "38630",
         "ecs.version": "1.0.0-beta2",
         "event.dataset": "nats.log",
         "event.module": "nats",
         "fileset.name": "log",
         "input.type": "log",
         "log.level": "trace",
         "log.offset": 359,
-        "message": "172.18.0.1:38630 - cid:1 - ->> [CONNECT {\"verbose\":false,\"pedantic\":false,\"tls_required\":false,\"name\":\"NATS Benchmark\",\"lang\":\"go\",\"version\":\"1.7.0\",\"protocol\":1,\"echo\":true}]",
+        "message": "{\"verbose\":false,\"pedantic\":false,\"tls_required\":false,\"name\":\"NATS Benchmark\",\"lang\":\"go\",\"version\":\"1.7.0\",\"protocol\":1,\"echo\":true}",
+        "nats.log.client.id": "1",
```

Comment:
@webmat This one is not very unique but perhaps something we could use in ECS ;-)

Reply:
You mean `client.id`?

Reply:
yes, kind of fits well with all the other `*.id` fields we have